src/Controls/tests/DeviceTests/Elements/Editor/EditorTests.cs

```diff
@@ -70,5 +70,22 @@ public async Task TextTransformUpdated(string text, TextTransform transform, str
 			var platformText = await GetPlatformText(handler);
 			Assert.Equal(expected, platformText);
 		}
+
+		[Fact]
+		public async Task MaxLengthIsReadOnlyValueTest()
+		{
+			Editor editor = new Editor();
+			editor.MaxLength = 0;
+
+			await InvokeOnMainThreadAsync(() =>
+			{
+				var handler = CreateHandler<EditorHandler>(editor);
+				var platformControl = GetPlatformControl(handler);
+				editor.MaxLength = 10;
+#if WINDOWS
+				Assert.False(platformControl.IsReadOnly);
+#endif
```

Comment:
Why only windows?

Reply:
We can make it IsReadOnly for all platforms if we want for consistency - but I didn't do this on Android and iOS because it wasn't necessary.

The reason we added the IsReadOnly logic to Windows is because when MaxLength > 0, it still allows you to type more text into the box. On Android and iOS, when MaxLength > 0, you can't type anything into the box even if IsReadOnly=false